docs/usage.md

```diff
@@ -93,6 +93,15 @@ within the `Reflector`s. The library comes bundled with the following
 
  * `AggregateSourceLocator` - a combination of multiple `SourceLocator`s which
     are hunted through in the given order to locate the source.
+    
+ * `FileSystemIteratorSourceLocator` - uses `SPL FileSystemIterator` to iterate over files
```

Comment:
Just wondering about naming here to make it clearer what the use cases are...

Am I correct in thinking:
 * `FileSystemIteratorSourceLocator` - iterates all files in a single directory (and descends into child directories)
 * `DirectorySourceLocator` iterates a list of directories (and their children), basically an aggregate version of `FileSystemIteratorSourceLocator`?

If that is the case, I'd propose renaming them to:

 * `FileSystemIteratorSourceLocator` => `SingleDirectorySourceLocator`
 * `DirectorySourceLocator` => `MultipleDirectoriesAggregateSourceLocator`

Currently, it's not 100% clear (looking *just* at the naming) what either of these do IMO.